benchmark/inclusion/inclusion.cc

```diff
@@ -480,6 +477,11 @@ namespace aspect
       Vector<float> cellwise_errors_ul2 (this->get_triangulation().n_active_cells());
       Vector<float> cellwise_errors_pl2 (this->get_triangulation().n_active_cells());
 
+      double u_l1;
```

Comment:
is there a reason you move the declaration of these variables up here? The code would shorter if you just do
```
double u_l1 = Utilities:: ...
```